src/middlewared/middlewared/plugins/interface/configure.py

```diff
@@ -144,14 +146,17 @@ def configure(self, data, aliases, wait_dhcp=False, options=None):
             if has_ipv6 and str(addr.address).startswith('fe80::'):
                 continue
             if addr not in addrs_database:
+                # keepalived is responsible for removing the VIP on SCALE HA
+                if vrrp_vip is not None and str(addr.address) in vrrp_vip:
```

Comment:
Should that be an "==" ?
10.0.0.1 in 10.0.0.10 would be True

Reply:
Further testing shows that keepalived doesn't care if middlewared or itself adds/removes the IP so I've removed this check.